contrib/scrooge/src/python/pants/contrib/scrooge/tasks/thrift_linter.py

```diff
@@ -21,7 +23,7 @@ class ThriftLintError(Exception):
   """Raised on a lint failure."""
 
 
-class ThriftLinter(NailgunTask):
+class ThriftLinterBase(NailgunTask):
   """Print linter warnings for thrift files."""
```

Comment:
Should we push this docstring down to the subclasses? We currently don't look at super classes docstrings when determining the task help text, so both of the sub tasks have no description.

Reply:
Good idea. Done.